Tests/TestingTests/AttachmentTests.swift

```diff
@@ -0,0 +1,319 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+@testable @_spi(Experimental) @_spi(ForToolsIntegrationOnly) import Testing
+private import _TestingInternals
+
+@Suite("Attachment Tests")
+struct AttachmentTests {
+  @Test func saveValue() {
```

Comment:
Should there be an expectation added to this test? What is it validating precisely?

Reply:
That the code to make an attachment compiles and runs (although not necessarily that it actually does anything.)

I intend to extend it later as more stuff lands.